pkg/providers/tinkerbell/hardware/yaml.go

```diff
@@ -84,81 +70,21 @@ func (yw *TinkerbellManifestYAML) write(data []byte) error {
 	return nil
 }
 
+// TODO(chrisdoherty4) Patch these types so we can generate yamls again with the new Hardware
+// and BaseboardManagement types.
+
 func marshalTinkerbellHardwareYAML(m Machine) ([]byte, error) {
-	return yaml.Marshal(
-		tinkv1alpha1.Hardware{
-			TypeMeta: metav1.TypeMeta{
-				Kind:       hardwareKind,
-				APIVersion: tinkerbellApiVersion,
-			},
-			ObjectMeta: metav1.ObjectMeta{
-				Name:      m.Hostname,
-				Namespace: constants.EksaSystemNamespace,
-				Labels: map[string]string{
-					clusterctlv1alpha3.ClusterctlMoveLabelName: "true",
-				},
-			},
-			Spec: tinkv1alpha1.HardwareSpec{
-				ID:     m.ID,
-				BmcRef: formatBMCRef(m),
-			},
-		},
-	)
+	return yaml.Marshal(tinkv1alpha1.Hardware{})
 }
 
 func marshalTinkerbellBMCYAML(m Machine) ([]byte, error) {
 	return yaml.Marshal(
-		pbnjv1alpha1.BMC{
```

Comment:
These will be updated. They currently only feature in the `generate` command so will circle round when we have a functional `create` cmd.